From 5f122cea440d7187faf0148f5ffac2ed1d03ae0f Mon Sep 17 00:00:00 2001
From: Yu Watanabe <watanabe.yu+github@gmail.com>
Date: Tue, 6 Mar 2018 16:02:44 +0900
Subject: [PATCH] rpm: add missing '-p <lua>' in trigger script (#8367)

Follow-up for 32a00a9c097cf04ec2b0fcbf9b73eba188318424 (#8090).

(cherry picked from commit 694d57655cbf11864bb9e202a901021c7e568ba7)
---
 src/core/triggers.systemd.in | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/src/core/triggers.systemd.in b/src/core/triggers.systemd.in
index c582d40977..4ebb4e7795 100644
--- a/src/core/triggers.systemd.in
+++ b/src/core/triggers.systemd.in
@@ -84,7 +84,7 @@ if posix.access("/run/systemd/system") then
     end
 end
 
-%transfiletriggerin -P 100500 --  @tmpfilesdir@
+%transfiletriggerin -P 100500 -p <lua> -- @tmpfilesdir@
 -- This script will process files installed in @tmpfilesdir@ to create
 -- tmpfiles automatically. The priority is set such that it will run
 -- after the sysusers file trigger, but before any other triggers.
@@ -97,7 +97,7 @@ if posix.access("/run/systemd/system") then
     end
 end
 
-%transfiletriggerin --  @udevhwdbdir@
+%transfiletriggerin -p <lua> -- @udevhwdbdir@
 -- This script will automatically invoke hwdb update if files have been
 -- installed or updated in @udevhwdbdir@.
 if posix.access("/run/systemd/system") then
@@ -109,7 +109,7 @@ if posix.access("/run/systemd/system") then
     end
 end
 
-%transfiletriggerin --  @catalogdir@
+%transfiletriggerin -p <lua> -- @catalogdir@
 -- This script will automatically invoke journal catalog update if files
 -- have been installed or updated in @catalogdir@.
 if posix.access("/run/systemd/system") then
@@ -121,7 +121,7 @@ if posix.access("/run/systemd/system") then
     end
 end
 
-%transfiletriggerin -- @udevrulesdir@
+%transfiletriggerin -p <lua> -- @udevrulesdir@
 -- This script will automatically update udev with new rules if files
 -- have been installed or updated in @udevrulesdir@.
 if posix.access("/run/systemd/system") then
@@ -133,7 +133,7 @@ if posix.access("/run/systemd/system") then
     end
 end
 
-%transfiletriggerin -- @sysctldir@
+%transfiletriggerin -p <lua> -- @sysctldir@
 -- This script will automatically apply sysctl rules if files have been
 -- installed or updated in @sysctldir@.
 if posix.access("/run/systemd/system") then
@@ -145,7 +145,7 @@ if posix.access("/run/systemd/system") then
     end
 end
 
-%transfiletriggerin -- @binfmtdir@
+%transfiletriggerin -p <lua> -- @binfmtdir@
 -- This script will automatically apply binfmt rules if files have been
 -- installed or updated in @binfmtdir@.
 if posix.access("/run/systemd/system") then
-- 
2.17.0

